scipy/sparse/tests/test_base.py

```diff
@@ -4354,8 +4358,40 @@ def _insert_explicit_zero(self, M, i, j):
         M[i,j] = 0
         return M
 
-
 class _NonCanonicalCSMixin(_NonCanonicalCompressedMixin):
+    def _arg1_for_noncanonical(self, M, sorted_indices=False):
+        """Return non-canonical constructor arg1 equivalent to M"""
+        data, indices, indptr = _same_sum_duplicate(M.data, M.indices,
+                                                    indptr=M.indptr)
+        if not sorted_indices:
+            for start, stop in izip(indptr, indptr[1:]):
+                indices[start:stop] = indices[start:stop][::-1].copy()
+                data[start:stop] = data[start:stop][::-1].copy()
+        return data, indices, indptr
+
+    def test_getelement(self):
+        def check(dtype, sorted_indices=False):
```

Comment:
I'd use a regular (non-keyword) argument here, because this is only getting called by the test runner anyway.